src/python/grpcio_tests/tests/unit/_channel_fd_test.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2016 gRPC authors.
```

Comment:
I suspect this line was copied and pasted?

Reply:
Yep. I've updated it now.